src/ColorSpace.d.ts

```diff
@@ -16,6 +16,7 @@ export interface Format {
 	id?: string | undefined;
```

Comment:
```suggestion
	id?: string | undefined;
	ids?: string[] | undefined;
```

The zyz-d65 space uses the `ids` property.